tests/remote_test.sh

```diff
@@ -127,6 +127,47 @@ function test_populate_remote_info()
   }
 }
 
+function test_is_ssh_connection_configured()
+{
+  local remote='test_remote'
+  local user='test_user'
+  local port='22'
+  local flag='TEST_MODE'
+
+  cd "$TEST_PATH" || {
+    fail "($LINENO) It was not possible to move to temporary directory"
+    return
+  }
```

Comment:
Before that, add something like:
```
local current_path="$PWD"
... to the end of this test ...
  cd "$current_path" || {
    fail "($LINENO) It was not possible to move to the original path"
    return
  }
```

In some point, we will need to refactor this test to use this cd in and out approach.